lib/tasks/translation.rake

```diff
@@ -1,4 +1,5 @@
 require "rails_translation_manager"
+require 'i18n/tasks/cli'
```

Comment:
This change could be fixed up to an earlier commit (i.e. the commit where these rake tasks were first introduced). And also switch to double quotes for consistency with the line above.

Reply:
@ChrisBAshton have moved it to the first commit